README.md

```diff
@@ -5,38 +5,58 @@
   </picture>
 </p>
 
-[![Join Slack](https://img.shields.io/badge/SkyPilot-Join%20Slack-blue?logo=slack)](https://join.slack.com/t/skypilot-org/shared_invite/zt-1i4pa7lyc-g6Lo4_rqqCFWOSXdvwTs3Q)
-![pytest](https://github.com/skypilot-org/skypilot/actions/workflows/pytest.yml/badge.svg)
-[![Documentation Status](https://readthedocs.org/projects/skypilot/badge/?version=latest)](https://skypilot.readthedocs.io/en/latest/?badge=latest)
+<p align="center">
+  <a href="https://skypilot.readthedocs.io/en/latest/"> 
+    <img alt="Documentation" src="https://readthedocs.org/projects/skypilot/badge/?version=latest">
+  </a>
+  
+  <a href="https://github.com/skypilot-org/skypilot/releases"> 
+    <img alt="GitHub Release" src="https://img.shields.io/github/release/skypilot-org/skypilot.svg">
+  </a>
+  
+  <a href="https://join.slack.com/t/skypilot-org/shared_invite/zt-1i4pa7lyc-g6Lo4_rqqCFWOSXdvwTs3Q"> 
+    <img alt="Join Slack" src="https://img.shields.io/badge/SkyPilot-Join%20Slack-blue?logo=slack">
+  </a>
+  
+</p>
+
+
+<h3 align="center">
+    Run jobs on any cloud, easily and cost effectively
+</h3>
 
 SkyPilot is a framework for easily running machine learning workloads[^1] on any cloud. 
 
-Use the clouds **easily** and **cost effectively**, without needing cloud infra expertise.
+SkyPilot abstracts away cloud infra burden:
+- Launch jobs & clusters on any cloud (AWS, Azure, GCP)
+- Find availability across zones/regions/clouds
+- Queue jobs & use cloud object stores
 
-_Ease of use_
-* **Run existing projects on the cloud** with zero code changes
-* Use a **unified interface** to run on any cloud, without vendor lock-in (currently AWS, Azure, GCP)
-* **Queue jobs** on one or multiple clusters
-* **Automatic failover** to find scarce resources (GPUs) across regions and clouds
-* **Use datasets on the cloud** like you would on a local file system 
+SkyPilot cuts your cloud costs:
+* [Managed Spot](https://skypilot.readthedocs.io/en/latest/examples/spot-jobs.html): **3x cost savings** using spot VMs, with auto-recovery from preemptions
```

Comment:
Surprisingly some people are not familiar with Spot VM. Thought it might be a good idea to link pages like https://aws.amazon.com/ec2/spot/ or https://cloud.google.com/spot-vms.
Reason: I asked a UCLA CS Phd friend and he's not aware of this type of VM. and surprised about the cost saving.